fs/testdata/empty_dir.ts

```diff
@@ -1,6 +1,6 @@
 import { emptyDir } from "../empty_dir.ts";
 
-emptyDir(Deno.args[0])
```

Comment:
As for this change, there is an indeed case that it may `undefined` for the `Deno.args[0]`, but the `empty` function has wrapped in `try/catch` block. On the other hand, it's hard to say exactly what type is it on the runtime, it may `string` or `URL`. Thus I think it's ok to use `!` operator.